src/core/shared_state/currentUser.ts

```diff
@@ -17,25 +18,34 @@ const publicUser: CurrentUser = {
   id: PUBLIC_USER_ID,
 };
 
+// defaults, not provided by api/missing in profile
```

Comment:
comment is not clear for me, try
```suggestion
// default values provided for fields that were missed in the api call
```